package.json

```diff
@@ -21,7 +21,8 @@
   },
   "homepage": "https://github.com/rangle/augury",
   "scripts": {
-    "build": "npm run webpack --colors --display-error-details --display-cached",
+    "build": "webpack --colors --display-error-details --display-cached",
```

Comment:
If just "webpack" is safe for `npm run webpack`, then "webpack --colors --display-error-details --display-cached" is safe for `npm run build`